### DISCUSSION TITLE
Lichess<title>

### DISCUSSION THREAD

**Discussion by Whasanike** [community/community#39781] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Two things happened in games on lichess in which I was contending. The most recent was an inability to input a move until I had refreshed the board several times, meanwhile the position on the board had not changed while it was still my turn to move. Trying to select the piece I eventually was allowed to move the clock was still running. 
The second weird thing that happened in the previous game was I had achieved a totally winning position and the game was truncated by the server in my favor with the message "cheating detected". If my opponent had been cheating, his play was lousy, so what kind of cheating results in a losing position? This has to be result of a error in the lichess program IMHO.

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/lichess-org/lila? In that case you should look for support information *in that repository* (usually in a file called README or SUPPORT). I notice the repository has discussions enabled, too, so maybe asking there would help. :slightly_smiling_face:

**Comment by PaquitoFIMO** [comment]
Sorry for my bad english, 
j'utilise GOOGLE TRANSLATE
VOIR IMAGE en fin de message
Look picture in end this message.


Hello,
For two days, or 3, it is impossible to download some pgns on tournaments, example here:

https://lichess.org/api/tournament/E3X0eqQ3/games

There is a network error:

Secure Connection Failed
An error occurred while connecting to lichess.org.
    The page you are trying to view cannot be displayed because the authenticity of the data received cannot be verified.
    Please contact the website owners to let them know about this issue.
And when downloading I get a "NETWORK FAILURES" error

Cordially

Cordialement
PaquitoFIMO sur lichess

MESSAGE ORIGINAL
Bonjour,
Depuis deux jours, ou 3, il est impossible de télécharger certains pgns sur les tournois, exemple ici : 

https://lichess.org/api/tournament/E3X0eqQ3/games

Il y a une erreur réseau : 

Échec de la connexion sécurisée
Une erreur est survenue pendant une connexion à lichess.org.
    La page que vous essayez de consulter ne peut pas être affichée car l’authenticité des données reçues ne peut être vérifiée.
    Veuillez contacter les propriétaires du site web pour les informer de ce problème.
Et lors du téléchargement, j'ai une erreur "ÉCHECS RÉSEAU"

Cordialement
PaquitoFIMO sur lichess

![2022_11_26_erreur_lichess](https://user-images.githubusercontent.com/119107474/204085676-9515b3d7-6251-4ef0-afbf-0f45fc92ffe3.jpg)
